### PR TITLE
[INLONG-8093][Manager] Add inlong tenant related APIs

### LINKED ISSUE #8093: [Feature][Manager] Add inlong tenant related APIs
### Description

parent #7914 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
- Fixes #8093

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by baomingyu — APPROVED** [review]
+1

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      <if test="version != null">
+        version,
+      </if>
+    </trim>
+    <trim prefix="values (" suffix=")" suffixOverrides=",">
+      <if test="id != null">
+        #{id,jdbcType=INTEGER},
+      </if>
+      <if test="name != null and name != ''">
+        name,
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml:75]: #{name,jdbcType=VARCHAR},

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      </if>
+      <if test="description != null and description != ''">
+        description,
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml:78]: #{description,jdbcType=VARCHAR},

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      </if>
+      <if test="isDeleted != null">
+        #{isDeleted,jdbcType=INTEGER},
+      </if>
+      <if test="creator != null and creator != ''">
+        creator,
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml:84]: #{creator,jdbcType=VARCHAR},

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      </if>
+      <if test="modifier != null and modifier != ''">
+        modifier,
```
Comment (fuweng11) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongTenantEntityMapper.xml:87]: #{modifier,jdbcType=VARCHAR},

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    public PageInfo<InlongTenantInfo> listByCondition(InlongTenantPageRequest request) {
+        PageHelper.startPage(request.getPageNum(), request.getPageSize());
+        Page<InlongTenantEntity> entityPage = inlongTenantEntityMapper.selectByCondition(request);
+        return entityPage
+                .toPageInfo(entity -> CommonBeanUtils.copyProperties(entity, InlongTenantInfo::new));
+    }
+
+    @Override
+    public Boolean update(InlongTenantRequest request) {
+        InlongTenantEntity exist = inlongTenantEntityMapper.selectByName(request.getName());
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java:77]: Whether the tenant name can be changed？

Comment (vernedeng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantServiceImpl.java:77]: If somebody change the tenant name, all the related resources needs to be changed also. Hence I don't think it's a good idea to allow tenant owner to change tenant name.

**Review by dockerzhang — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Pull request merged**